content/app/development/ux/texts/_index.en.md

```diff
@@ -113,6 +113,40 @@ The texts are stored in `App/config/texts`
 
 ![Altinn Studio Repos](edit-texts-in-repos.png "Change texts in Altinn Studio Repos")
 
+## Change dafault texts and error messages in an application
+
+It is possible to change default texts and error messages that is displayed in the app. 
+Here are the id:s with its default value in [English](https://github.com/Altinn/app-frontend-react/blob/v3/src/language/texts/en.ts), 
+[Norwegian Bokmål](https://github.com/Altinn/app-frontend-react/blob/v3/src/language/texts/nb.ts) and [Nynorsk](https://github.com/Altinn/app-frontend-react/blob/v3/src/language/texts/nn.ts) 
+
+Those default texts that has a number needs to be handled in a different way. For exampel `file_uploader_validation_error` displays an error message if 
+the user tries to continiue to the next page even though it is required to add one attachment. This default error message 
+will be displayed "You need to upload 1 attachement(s) to continue'.
```

Comment:
```suggestion
Those default texts that has a number needs to be handled in a different way. For example `file_uploader_validation_error` displays an error message if it is required to add one attachment. This default error message will be displayed as "You need to upload 1 attachement(s) to continue'.
```

The feature that stops the user from navigating to the next page is unrelated to this, and it can be shown without the app having that feature enabled, so it would't be fitting to mix that in here.